test/openAPI/features/support/helpers/helpers.js

```diff
@@ -1,4 +1,45 @@
 module.exports = {
   localhost: 'http://localhost:3333/',
   defaultResponseTime: 15000,
+  statusEmailEndpoint: 'status/email',
+  contentTypeHeader: {
+    key: 'content-type',
+    value: 'application/json; charset=utf-8',
+  },
+  allowHeaderPost: {
+    key: 'allow',
+    value: 'POST',
+  },
+  allowHeaderGet: {
+    key: 'allow',
+    value: 'GET',
+  },
+  acceptHeader: {
+    key: 'Accept',
+    value: 'application/json',
+  },
+  statusEmailResponseSchema: {
+    type: 'object',
+    properties: {
+      scheduled: { type: 'number' },
+      processing: { type: 'number' },
+      deferred: { type: 'number' },
+      bounced: { type: 'number' },
+      'not sent': { type: 'number' },
+      delivered: { type: 'number' },
+    },
+  },
+  doArraysHaveTheSameElements: (array1, array2) => {
+    if (array1.length === array2.length) {
+      return array1.every(element => {
+        if (array2.includes(element)) {
+          return true;
+        }
+
+        return false;
+      });
+    }
+
+    return false;
+  },
```

Comment:
I see here a place for improvement. If I understand good you want to compare two arrays - one is with request params and second is your response, am I correct? 
They need do have the same elements inside but they are not sorted so you need to check if second array includes element from first one. 
You can write less code if you first sort the arrays and then just change this code to the 
```
doArraysHaveTheSameElements: (array1, array2) =>
  array1.length === array2.length &&
  array1.every((element, index) => element === array2[index]);
```
Then you can change the name of the function, because it return boolean, so `areArraysTheSame`

Here is a lot of examples: https://www.freecodecamp.org/news/how-to-compare-arrays-in-javascript/